### ISSUE TITLE
Too few bins? 

### ISSUE THREAD

**Issue by kwantn** [esrud/GONE#50]
Hi Armando and all, 

The provided example.ped and .map worked beautifully. However with my own data set, I am constantly facing the error 'Too few bins' despite altering to even NGEN=200, NBIN=200. I am working with DArTseq reduced representation SNPs, and upon filtering I have 16,194 SNPs (i.e. 8097 loci). This is the number of loci on the 20 chromosome.

![image](https://github.com/user-attachments/assets/c6ac6ab5-0ef6-451a-8716-de71cabe5ddd)

I attached my .bed and .map here. I hope this is not due to insufficient SNPs for the recombination bins.

Cheers kwan

**Comment by armando-caballero** [comment]
Yes, I think there are too few loci within each chromosome to make an enough number of bins.
Try other combinations of NGEN and NBIN.
Best wishes,
Armando
